test/test.js

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env node
 "use strict";
 
-import fs from "fs";
-import * as commonmark from "../lib/index.js";
+// Using commonjs here so CI can run the test file on older Node.js versions
```

Comment:
Note this comment, which was clobbered by 435f7b8aa82eb692c9ca14ffa4768703578359d1.

Until we drop support for pre-ESM Node.js versions, the things that we expect Node.js to run---the bin script, the test runner, the benchmark runner---should remain CommonJS.